openml/extensions/sklearn/run_functions.py

```diff
@@ -0,0 +1,453 @@
+from collections import OrderedDict
+import json
+import time
```

Comment:
Also, as according to git all functions here are `new` I did not have a thorough look at them as I assumed they will still behave the same. 

Did any of the functions changed / do you want me to look at any of them in depth? Otherwise I think we should rely on the unit tests.

Reply:
No, I think I didn't touch the functions themselves, only the signatures (adding type hints).